src/main/java/com/katelocate/menugenerator/recipe/Recipe.java

```diff
@@ -6,7 +6,6 @@ public record Recipe(
         Integer id,
         @NotEmpty
         String title,
-        @NotEmpty
```

Comment:
Is it enough to remove just this validator but noth the one above?

Reply:
Yes, that is because @NotEmpty annotation is used for collections and similar objects, I guess. It checks if the value is not null and if its length is greater than zero. My recipe class lacks the length property, I think.